core/src/tui_core/styles/stylesheet.rs

```diff
@@ -73,3 +73,63 @@ impl Stylesheet {
     }
   }
 }
+
+/// Macro to make building `StyleSheet` easy
+///
+/// Use case:
+/// ```rust
+/// use crossterm::event::*;
+/// use r3bl_rs_utils_core::{CommonResult, Stylesheet};
+///
+/// fn create_stylesheet() -> CommonResult<Stylesheet> {
+///   throws_with_return!({
+///     stylesheet! {
+///         style! {
+///           id: style1
+///           margin: 1
+///           color_bg: Color::Rgb { r: 55, g: 55, b: 248 }
+///         },
+///         vec![
+///             style! {
+///                 id: style1
+///                 margin: 1
+///                 color_bg: Color::Rgb { r: 55, g: 55, b: 248 }
+///             },
+///             style! {
+///                 id: style2
+///                 margin: 1
+///                 color_bg: Color::Rgb { r: 85, g: 85, b: 255 }
+///             },
+///         ]
+///     }
+///   })
+/// }
+/// ```
+/// Note that this operation may panic if the styles could not be added to the Stylesheet
+///
+#[macro_export]
+macro_rules! stylesheet {
+    ($($style:expr),*) => {
+        {
+            trait AddStyle {
+                #[inline]
```

Comment:
Remove `#[inline]`. It actually triggers a clippy warning.
https://rust-lang.github.io/rust-clippy/master/index.html#inline_fn_without_body

1. You can also run clippy on file save. By configuring IDEA in this manner: https://github.com/intellij-rust/intellij-rust/issues/4756#issuecomment-565907188 
2. Or you can run it by executing the `cargo clippy` command. You can even run `cargo clippy --fix --allow-dirty --allow-staged` and it will apply some fixes that it can automatically.